tests/test_split_on_grid.py

```diff
@@ -55,13 +55,32 @@ class TestSplitOnGrid(unittest.TestCase):
     @parameterized.expand(
         [TEST_CASE_0, TEST_CASE_1, TEST_CASE_2, TEST_CASE_3, TEST_CASE_4, TEST_CASE_5, TEST_CASE_6, TEST_CASE_7]
     )
-    def test_split_pathce_single_call(self, input_parameters, img, expected):
+    def test_split_patch_single_call_numpy(self, input_parameters, img, expected):
```

Comment:
I think you can combine the numpy input and tensor input tests, including GPU / CPU tenors, etc. Just refer to:
https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_as_channel_first.py#L22

Thanks.

Reply:
That's nice! Thanks